websites/A/AnimeDigitalNetwork/presence.ts

```diff
@@ -37,17 +36,16 @@ presence.on("UpdateData", async () => {
         delete data.endTimestamp;
       }
 
-      if (t !== null) {
+      if (title !== null) {
         presence.setActivity(data);
       }
     }
   } else if (document.location.pathname.includes("video") && !video) {
-    const t = document
+    const title = document
         .querySelector(
-          "#root > div > div > div.sc-qXJnB.ijvXfJ > div > div > div.sc-pAArZ.idjhKu > div:nth-child(2) > div > div > div > div.sc-jwDLgl.jjQkhb > div > h1"
+          "#root > div > div > div.sc-pbWVv.hTvDIL > div > div:nth-child(1) > div.sc-jWJfXU.sc-kbKFCX.ibhpg > div:nth-child(1) > div > div > h1 > a"
         )
-        .textContent.toLowerCase(),
-      title = t.charAt(0).toUpperCase() + t.slice(1),
+        .textContent,
       data = {
         details: "Browsing...",
```

Comment:
well I think it should be:
```suggestion
        details: "Looking at",
```